lib/modules/manager/bazel-module/bazel-dep.ts

```diff
@@ -0,0 +1,23 @@
+import is from '@sindresorhus/is';
+import { BazelDatasource } from '../../datasource/bazel';
+import type { PackageDependency } from '../types';
+import { RecordFragment, StringFragment, ValueFragment } from './fragments';
+
+export function toPackageDependency(
+  value: ValueFragment
+): PackageDependency | null {
+  const record = RecordFragment.safeAs(value);
+  if (!record) {
+    return null;
+  }
+  const { rule, name, version } = record.children;
+  if (is.falsy(rule) || is.falsy(name) || is.falsy(version)) {
```

Comment:
why using falsy? which values can occur?

Reply:
I have no idea what I was thinking. 😳 I updated this section to look like the implementation that will appear in a later PR.